corehq/apps/app_manager/util.py

```diff
@@ -153,18 +153,17 @@ def change_xmlns(xform, replacing):
 
     form.source = xml
 
-    # For registration forms, assume that the first question is the case name
+    # Assume that the first question is the case name
     # unless something else has been specified
-    if toggles.APP_MANAGER_V2.enabled(app.domain):
```

Comment:
Yes, intentional. It didn't seem worth the hassle to get ahold of the current user at all the places where `save_xform` is called. So this will change all apps, but in a minor way.

Not sure why I changed that comment, will change that back.